### PR TITLE
update doc guide

### PR DESCRIPTION
**Pull request by jason-jszhang**

### CODE REVIEW

File: src/en/guide/get_started/installation.md [hunk 1]
```diff
@@ -26,8 +26,17 @@ pip install metagpt
 
 ### Install submodules
 
-- RAG, `pip install metagpt[rag]`. On macOS/zsh: `pip install 'metagpt[rag]'`
-
+- RAG, `pip install metagpt[rag]`.
```
Comment (better629) [inline src/en/guide/get_started/installation.md:29]: Use pip install 'metagpt[rag]' and put it in quotes, as well

File: src/en/guide/tutorials/customize_llms_for_roles_or_actions.md [hunk 2]
```diff
@@ -74,3 +74,57 @@ team = Team(investment=10.0, env=env, roles=[A, B, C])
 asyncio.run(team.run(idea="Topic: climate change. Under 80 words per message.", send_to="A", n_round=3))
 # await team.run(idea="Topic: climate change. Under 80 words per message.", send_to="A", n_round=3) # If running in Jupyter Notebook, use this line of code
```
Comment (better629) [inline src/en/guide/tutorials/customize_llms_for_roles_or_actions.md:75]: Missing multiple LLMS in config2.yaml

File: src/zh/guide/get_started/configuration/llm_api_configuration.md [hunk 3]
```diff
@@ -92,7 +92,7 @@ llm:
   base_url: 'YOUR_AZURE_BASE_URL'
   api_key: 'YOUR_API_KEY'
   api_version: 'YOUR_API_VERSION' # such as '2024-03-01-preview'
-  model: 'YOUR_MODEL_NAME' # or YOUR_DEPLOYMENT_NAME
+  model: 'YOUR_MODEL_      NAME' # or YOUR_DEPLOYMENT_NAME
```
Comment (better629) [inline src/zh/guide/get_started/configuration/llm_api_configuration.md:95]: multi spaces??

File: src/en/guide/tutorials/customize_llms_for_roles_or_actions.md (hunk 2 above)
Comment (jason-jszhang) [reply src/en/guide/tutorials/customize_llms_for_roles_or_actions.md:75]: Multiple LLMS configurations cannot be configured in the same config file.  Therefore, you need to configure it in a new custom configuration file, such as gpt-4.yaml. Then use gpt4 = Config.from_home("gpt-4.yaml") to load the configuration.

**Pull request merged**